src/Model/Serializer/ValueObject/Calendar/DaysDenormalizer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Model\Serializer\ValueObject\Calendar;
+
+use CultuurNet\UDB3\Model\ValueObject\Calendar\OpeningHours\Day;
+use CultuurNet\UDB3\Model\ValueObject\Calendar\OpeningHours\Days;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+
+final class DaysDenormalizer implements DenormalizerInterface
+{
+    public function denormalize($data, $class, $format = null, array $context = []): Days
+    {
+        $days = [];
```

Comment:
If you want to be fancy you can replace this entire function with:
`return new Days(...array_map(fn($day) => new Day($day), $data));`

Reply:
Nice!